### ISSUE TITLE
Using trigger property causes notifications to never show on Android.

### ISSUE THREAD

**Issue by Gusten** [katzer/cordova-plugin-local-notifications#1436]
## Your Environment

* Plugin version: 0.9.0-beta.1
* Platform: Android
* OS version: 8.0.0 (And emulated versions, see below)
* Device manufacturer / model: Huawei Nexus 6P
* Cordova version (```cordova -v```): 7.1.0
* Cordova platform version (```cordova platform ls```): 6.3.0
* Plugin config: default
* Ionic Version (if using Ionic): 3.18.0

I experience the same issue when emulating Android 7.1.1 and Android 6.0.0.

## Expected Behavior

Notification should show in status bar after 5 seconds.

## Actual Behavior

No notification is displayed at all.

## Steps to Reproduce

```javascript
cordova.plugins.notification.local.schedule({
  id: 1,
  text: "Test",
  trigger: { in: 5, unit: "second" }
});
```

## Context

Using the property trigger when scheduling a notification causes it to never show. If I remove the trigger property it works as intended. I also tried using trigger property with at instead of in but it did not make any difference. I also tried using different timespan units such as minute but still nothing. Furthermore, I tried different priority levels but no go.

**Update**: Downgrading to 0.9.0-beta.0 fixes the issue

## Debug logs

The only log I get is from when the "add" event is fired. Nothing happens after the 5 seconds(Same on device as well on both emulated versions I tried).

D/CordovaWebViewImpl: >>> loadUrl(javascript:cordova.plugins.notification.local.core.fireEvent("add",{"id":1,"text":"Test","trigger":{"in":5,"unit":"second","type":"calendar"},"actions":[],"attachments":[],"autoClear":true,"defaults":0,"foreground":false,"groupSummary":false,"launch":true,"led":true,"lockscreen":true,"number":0,"priority":0,"progressBar":{"enabled":false,"value":0,"maxValue":100,"indeterminate":false},"showWhen":true,"silent":false,"smallIcon":"res:\/\/icon","sound":true,"title":"","vibrate":false,"wakeup":true,"meta":{"plugin":"cordova-plugin-local-notifications","version":"0.9-beta"}},{"event":"add","foreground":true,"queued":false,"notification":1}))

**Comment by Gusten** [comment]
If anyone else sees this. For me, this issue was solved by downgrading to 0.9.0-beta.0.

**Comment by katzer** [comment]
@Gusten I cant reproduce with Android 7 and latest master branch. Do you see any related messages in the log or stack traces?

**Comment by Gusten** [comment]
@katzer 
Thanks for your response.
I tried reproducing this issue today to give more information but didn't manage to. I'm guessing something went wrong in the build when I upgraded from 8.* to 9 beta1 and I somehow managed to fix it when I downgraded to beta 0.

Just to make sure I pulled your latest commits and they all seem to work as well. I apologize for the wrongly reported issue. I'm closing this issue.

Thanks for your work.
